test/bootstrap_test.go

```diff
@@ -65,3 +65,14 @@ func TestBootstrapError(t *testing.T) {
 		assert.Fail(t, "got weird error")
 	}
 }
+
+func TestBootstrapWithBadLogLevel(t *testing.T) {
+	gateway, err := testGateway.CreateGateway(t, map[string]interface{}{
+		"logger.level": "invalid",
+	}, &testGateway.Options{
+		TestBinary:  util.DefaultMainFile("example-gateway"),
+		ConfigFiles: util.DefaultConfigFiles("example-gateway"),
+	})
+	assert.Error(t, err, "got bootstrap err")
```

Comment:
Shouldn't we assert that the error we get is unknown log level?

Reply:
Unfortunately the error message gets munged by the test gateway, so it's hard to assert the error message.

Reply:
Added some unit tests as well in another file.